pmlc/rt/runtime_registry.h

```diff
@@ -47,7 +47,7 @@ using Factory = std::function<std::shared_ptr<Runtime>()>;
 // N.B. This function is NOT synchronized.  It is the caller's responsiblity to
 // ensure that other components are not concurrently accessing the system global
 // runtime map or registering other Factory instances.
-void registerFactory(llvm::StringRef id, Factory factory);
+void registerFactory(::llvm::StringRef id, Factory factory);
```

Comment:
Actually you could use `mlir::StringRef` here

Reply:
Turns out that doesn't work for `::llvm::StringMap`, but otherwise, it's great - thanks!  Done.